rest-api-spec/src/main/resources/rest-api-spec/test/get_source/40_routing.yml

```diff
@@ -26,7 +26,6 @@
  - do:
       get_source:
           index:   test_1
-          type:    test
```

Comment:
I think that in order to complete successfully, we'll need to have a `skip` section that limits these tests to 7.0.

We'll also want to retain the old typed versions of these tests, as we've done for the other deprecated APIs. For example, we can introduce a new file `41_routing_with_types` to continue to test the typed calls.